openapi/kas-fleet-manager-private.yaml

```diff
@@ -483,6 +483,15 @@ components:
           description: "Data plane cluster agent spec"
           type: object
           properties:
```

Comment:
@shawkins this is the spec change to the managedkafkaagent. It should corresponds to what's in  https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/pull/840 but I'd like to have another pair of eyes on this.

Reply:
Yes that matches.